main.tf

```diff
@@ -156,7 +156,8 @@ module "ecs-service-cloudwatch-dashboard" {
  * Create RDS root password
  */
```

Comment:
Maybe include a comment about `override_special` having all the defaults except for `@`